spvc/test/run_spirv_cross_tests.py

```diff
@@ -190,38 +193,38 @@ def lookup(table, filename):
             break
     return haystack
 
+
 shader_models = (
     'sm60', '60',
     'sm51', '51',
     'sm30', '30',
-    ''    , '50',
+    '',     '50',
 )
 msl_standards = (
-    'msl2' , '20000',
+    'msl2',  '20000',
     'msl21', '20100',
     'msl11', '10100',
-    ''     , '10200',
+    '',      '10200',
 )
 msl_standards_ios = (
-    'msl2' , '-std=ios-metal2.0',
+    'msl2',  '-std=ios-metal2.0',
     'msl21', '-std=ios-metal2.1',
     'msl11', '-std=ios-metal1.1',
     'msl10', '-std=ios-metal1.0',
-    ''     , '-std=ios-metal1.2',
+    '',      '-std=ios-metal1.2',
 )
 msl_standards_macos = (
-    'msl2' , '-std=macos-metal2.0',
+    'msl2',  '-std=macos-metal2.0',
     'msl21', '-std=macos-metal2.1',
     'msl11', '-std=macos-metal1.1',
-    ''     , '-std=macos-metal1.2',
+    '',      '-std=macos-metal1.2',
 )
 
 
 # Test spvc producing MSL the same way SPIRV-Cross is tested.
 # There are three steps: compile input, convert to HLSL, check result.
 def test_msl(shader, filename, optimize):
```

Comment:
Method comment should describe the return values.

Reply:
Done